frontend/components/GravePageAssets.tsx

```diff
@@ -18,7 +18,7 @@ export default function GravePageAssets({
   const [error, setError] = useState<string>();
 
   useEffect(() => {
-    if (!graveVault && pageAccount) {
```

Comment:
do we want to remove this?

Reply:
yeah because pageAccount is not not optional so it's always there